_data/flux-commands.yaml

```diff
@@ -54,10 +54,17 @@ commands:
             command: flux jobs --no-header -o '{status}:{returncode}' ƒj4CgebBV
           - title: Show me the job ids in a particular state
             command: flux jobs --filter=pending,running,inactive -o "{id}"
+
+  - name: flux job
+    groups:
+      - name: Basic interaction with jobs
+        items:
           - title: Get the ID of the last submitted job
             command: flux job last
           - title: Get the output of the last submitted job
             command: flux job attach $(flux job last)
+          - title: Send a signal to a specific job
+            command: flux job kill --signal=SIGUSR1 ƒj4CgebBV
```

Comment:
When would someone use `flux job kill` with this flag vs `flux job raise`?

Reply:
its very common to program signal handlers to capture signals and do something in the program.  A very common use case is to capture some specific signal, clean up something, then exit.

Reply:
So when should we use the above vs raise or raiseall?

Reply:
raise/raiseall generate an exception, which may not generate a signal.  If you generate an exception with severity=0, then it will effectively cancel the job and (i think) send SIGTERM to the job, and that signal is not controllable.  So if you want to send specifically SIGTERM, `flux job raise --severity=0` and `flux cancel` might be viewed be as similar to `flux job kill --signal=SIGTERM <jobid>`.  But if a person wants to send any other signal, they have to use `flux job kill`.

Reply:
Okay gotcha - it sounds like for most users they would want the command here. I wanted to double check because this scenario feels similar to flux cancel vs flux job cancel - two different commands with a similar result (and I want to understand which we should be teaching).

Did you preview locally and are happy with the rendered page? That’s the last thing to check!

Reply:
i did not, not even sure how to render it. looking through configs i guess you're using jekyll?

Reply:
the rendering looks good to me!

Reply:
future note to self, do not waste time with jekyll on RHEL8, it's a dependency hellhole ...

Reply:
haha sorry about that - also I fell back asleep so apologies for the delay! I can't help being a sleep monster :P 

Let's get this merged!